clang/lib/Basic/Targets/OSTargets.cpp

```diff
@@ -215,8 +215,7 @@ static void addVisualCDefines(const LangOptions &Opts, MacroBuilder &Builder) {
 
     if (Opts.isCompatibleWithMSVC(LangOptions::MSVC2015)) {
       if (Opts.CPlusPlus23)
-        // TODO update to the proper value.
-        Builder.defineMacro("_MSVC_LANG", "202004L");
+        Builder.defineMacro("_MSVC_LANG", "202302L");
```

Comment:
STL mention setting _MSVC_LANG to 202600 in C++26, we should probably follow suite

Reply:
Good point. My only concern is that Clang currently sets `__cplusplus` to 202400L in C++26 mode, so they wouldn't match which is weird. But maybe we could bump that to 202600 also?

Reply:
Clang and GCC both use `202400L`: https://godbolt.org/z/7er4doE6h -- maybe Microsoft wants to consider using the same value?

Reply:
This seems perfectly reasonable. I'll throw it over the fence.

Reply:
From a user perspective, having compilers align on a particular value in the range "greater than C++23's value, but less than C++26's final value" doesn't seem important. But from Clang's perspective, I can see how it would be simpler to avoid having to vary depending on whether you're imitating MSVC or not. I'll make this change for our implementation.

Reply:
I've updated the PR to set _MSVC_LANG to 202400L for C++26.